### ISSUE TITLE
Option Lists - Sorting

### ISSUE THREAD

**Issue by T1mClark** [formtools/core#233]
As nice as the drag n' drop reordering is, it is tedious if you have long lists. it would be nice if there was also an option to simply sort the entire list with a single click.

**Comment by benkeen** [comment]
Agreed. The functionality around the Option Lists is a little thing right now, this would be useful. Thanks!
